org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/RunnableDeviceServiceImpl.java

```diff
@@ -51,50 +51,50 @@
 
 @SuppressWarnings("rawtypes")
 public final class RunnableDeviceServiceImpl implements IRunnableDeviceService, IScanService {
-	
+
 	private static final Logger logger = LoggerFactory.getLogger(RunnableDeviceServiceImpl.class);
-	
+
 	/**
 	 * The default Malcolm Hostname can be injected by spring. Otherwise
 	 * this machine will be used for the malcolm host for instance 'beamline'-control.
 	 */
 	public static String defaultMalcolmHostname = null;
-	
+
 	/**
 	 * This service can not be present for some tests which run in OSGi
 	 * but mock the test laster.
 	 */
-	private static IScannableDeviceService deviceConnectorService;	
-	
+	private static IScannableDeviceService deviceConnectorService;
+
 	/**
 	 * This service must be present.
 	 */
 	private static IMalcolmService         malcolmService;
-	
+
 	/**
 	 * Map of device model class to device class.
 	 * NOTE This is not unmodifiable. Entries may be made after service create time. For instance
 	 * when the spring files are parsed.
 	 */
 	private static final Map<Class<?>, Class<? extends IRunnableDevice>> modelledDevices;
-	
+
 	/**
 	 * Map of device name to created device. Used to avoid
 	 * recreating non-virtual devices many times.
-	 * 
+	 *
 	 * TODO Should this be populated by spring?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)